v2/robotmk/tests/test_scheduler.rs

```diff
@@ -17,6 +17,7 @@ use tokio::{process::Command, time::timeout};
 use walkdir::WalkDir;
 
 #[tokio::test]
+#[ignore]
```

Comment:
This currently leads to the test also being skipped by our system tests

Reply:
Done:
https://github.com/elabit/robotmk/actions/runs/7195151926/job/19597218491?pr=454